agent/structs/config_entry_routes.go

```diff
@@ -0,0 +1,177 @@
+package structs
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/consul/acl"
+)
+
+// HTTPRouteConfigEntry manages the configuration for a HTTP route
+// with the given name.
+type HTTPRouteConfigEntry struct {
```

Comment:
Is this just a stub for where future http route details will get added?  Unlike the TCP route below there seems to be no route information in this struct currently.

Thats fine if the intention is to come in a future PR, I just wanted to ensure I wasn't missing some detail.

Reply:
Yes, Andrew is working on it. It's more complex than TCP routing because there are more configurations for filtering.